src/components/Products.js

```diff
@@ -0,0 +1,47 @@
+// import React, { Component } from 'react'
+
+//  export default class Items extends Component {
+//     render() {
+//         return (
+//             <div>
+
+//                 <h3>Item Name</h3>
+//                 <p>Price</p>
+//                 <button>Add to cart</button>
+//                 <p>Picture</p>
+//                 <p>Item Description</p>
+
+//             </div>
+//         )
+//     }
+// }
+
+import React from "react";
+
+const Products = (props) => {
+  //   console.log(props);
+  //   console.log(props.keyOfProducts[0].name);
+    let product = props.keyOfProducts[0];
```

Comment:
Looks like you left some debugging code in here! Console logs are always helpful, but don't forget about the devtools. They are far more useful because you can actually watch state/props change as they update.